### DISCUSSION TITLE
GitHub Student Education Pack (Constant Rejection)

### DISCUSSION THREAD

**Discussion by Amit-Shan** [community/community#52819] [GitHub Education]
Hello. I just started my Computer Science degree at the University of London. It's an online degree programme. I am unable to get my student status verified. My documents are accurate, but I keep getting rejected because my GPS location cannot be the same as my University location due to it being an exclusively Online Degree. The GitHub page that lets me start my application process also states that The University of London has no location. It lists "Worldwide" on the page I select my university email address.  None of my official enrollment documents list any details about my distance learning status. The university does not list that anywhere except on the generic website. My official Registration only states the University of London as my institute. How do I prove that this is a distance learning institution? Are there any other ways I can prove my student status. 

The email I get lists the following reasons for rejection:-

1. The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
2. You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

**Comment by samus-aran** [comment]
Hi @Amit-Shan 👋 

How did you get on? Are you still having issues applying?
